### DISCUSSION TITLE
Prodocs Theme

### DISCUSSION THREAD

**Discussion by pisieng** [GetPublii/Publii#1343] [General]
Hi there, I would like to buy a Prodocs Theme. But I wanna to check before : 
1) Can I create more than one main documentations on a website ?
2) Can I highlight some featured posts ?
Thank.

**Comment by sandrobilbeisi** [comment]
With Publii, you can create multiple websites even on the same server.
You can publish them in folders, side by side.
You can publish in different domains and subdomains.

**Comment by candidexmedia** [comment]
> Can I highlight some featured posts ?

Yep, give the demo a look: https://demo.getpublii.eu/themes/prodocs/

There's a "Featured Articles" section on the homepage.
